unrpyc.py

```diff
@@ -187,64 +188,133 @@ def worker(args, filename):
+
+    _ap.add_argument(
+        '-p',
+        '--processes',
+        dest='processes',
+        action='store',
+        type=int,
+        choices=list(range(1, _cc_num)),
+        default=_cc_num,
+        help="use the specified number or processes to decompile."
+        "Defaults to the amount of hw threads available minus one, disabled when muliprocessing "
+        "unavailable is.")
+
+    _ap.add_argument(
+        '-t',
+        '--translation-file',
+        dest='translation_file',
+        action='store',
+        default=None,
+        help="use the specified file to translate during decompilation")
+
+    _ap.add_argument(
+        '-T',
+        '--write-translation-file',
+        dest='write_translation_file',
+        action='store',
+        default=None,
+        help="store translations in the specified file instead of decompiling")
+
+    _ap.add_argument(
+        '-l',
+        '--language',
+        dest='language',
+        action='store',
+        default='english',
+        help="if writing a translation file, the language of the translations to write")
+
+    _ap.add_argument(
+        '--comparable',
+        dest='comparable',
+        action='store_true',
+        help="Only for dumping, remove several false differences when comparing dumps. "
+        "This suppresses attributes that are different even when the code is identical, such as "
+        "file modification times. ")
+
+    _ap.add_argument(
+        '--no-pyexpr',
+        dest='no_pyexpr',
+        action='store_true',
+        help="Only for dumping, disable special handling of PyExpr objects, instead printing them "
+        "as strings. This is useful when comparing dumps from different versions of Ren'Py. It "
+        "should only be used if necessary, since it will cause loss of information such as line "
+        "numbers.")
+
+    _ap.add_argument(
+        '--no-init-offset',
+        dest='init_offset',
+        action='store_false',
+        help="By default, unrpyc attempt to guess when init offset statements were used and insert "
+        "them. This is always safe to do for ren'py 8, but as it is based on a heuristic it can be "
+        "disabled. The generated code is exactly equivalent, only slightly more cluttered.")
+
+    _ap.add_argument(
+        '--register-sl-displayable',
+        dest="sl_custom_names",
+        type=str,
+        nargs='+',
+        help="Accepts mapping separated by '=', "
+        "where the first argument is the name of the user-defined displayable object, "
+        "and the second argument is a string containing the name of the displayable,"
+        "potentially followed by a '-', and the amount of children the displayable takes"
+        "(valid options are '0', '1' or 'many', with 'many' being the default)")
+
+    return _ap.parse_args()
+
+
+def main(args):
+    """Main execution..."""
+    # NOTE: Code for file/dir work refactored to pathlib usage
```

Comment:
That docstring adds literally nothing over the function name. Also, that comment should be a commit message.